src/steamship/agents/llms/replicate.py

```diff
@@ -0,0 +1,150 @@
+from typing import List, Optional
+
+from steamship import Block, File, PluginInstance, Steamship, Tag
+from steamship.agents.schema import LLM, ChatLLM, Tool
+from steamship.data import GenerationTag, TagKind
+from steamship.utils.dict_utils import remove_none
+
+PLUGIN_HANDLE = "replicate-llm"
+DEFAULT_MAX_TOKENS = 256
+
+LLAMA_2_70B = "replicate/llama-2-70b-chat"
+DOLLY_2_12B = "replicate/dolly-v2-12b"
+
+DEFAULT_MODEL = LLAMA_2_70B
+
+
+class Replicate(LLM):
+    """LLM that uses Steamship's Replicate plugin to generate completions.
+
+    The default model is Llama 2 70B.
+
+    See the Replicate plugin source code for more information:
+      https://github.com/steamship-plugins/replicate-llms
+    """
+
+    generator: PluginInstance
+    client: Steamship
+
+    def __init__(
+        self,
+        client,
+        model_name: str = None,  # The default in the plugin is LLAMA_2_70B.
+        temperature: float = None,
+        model_version: Optional[str] = None,
+        default_system_prompt: Optional[str] = None,
+        max_tokens: Optional[int] = None,
+        top_p: Optional[float] = None,
+        stop_sequences: Optional[str] = None,  # Should be a CSV string
+        *args,
+        **kwargs
+    ):
+        """Create a new instance.
+
+        Valid model names are presently:
+         - replicate/llama-2-70b-chat
+         - replicate/dolly-v2-12b
+        """
```

Comment:
```suggestion
        a16z-infra/llama-2-13b-chat
        """
```

Reply:
It probably would make sense to have an endpoint on the plugin that could report these rather than having to remember to come back here to update these, come to think of it.

Reply:
I haven't been following the evolution of that plugin, but wasn't the idea that *any* replicate LLM could be used? What is the reasoning behind the limited selection here?